javascript/extractor/src/com/semmle/js/parser/TypeScriptASTConverter.java

```diff
@@ -1582,8 +1582,10 @@ private Node convertMetaProperty(JsonObject node, SourceLocation loc) throws Par
     Position metaStart = loc.getStart();
     Position metaEnd =
         new Position(metaStart.getLine(), metaStart.getColumn() + 3, metaStart.getOffset() + 3);
```

Comment:
The magic constant `3` here seems to have been outdated, as it should presumably be 4 for `import.meta`.